### DISCUSSION TITLE
请问，支持页面同时存在暗水印和明水印吗

### DISCUSSION THREAD

**Discussion by Jess-2021** [zhensherlock/watermark-js-plus#224] [Q&A] [answered]
Hi, 请问是否有方法来实现同个页面上存在多种水印，有点类似多实例。

据我实践貌似不支持，看了源码貌似有个 `validateUnique` 方法来判断，我是否可以通过修改 `__WATERMARK__` 属性来达到目的效果？

**Comment by zhensherlock** [comment, marked as answer]
是不是可以在body下放一个水印，然后在某个div下再放个水印（用parent属性）

**Reply by Jess-2021** [reply]
嗯，是可以的，原来只判断同个dom层级下的水印。感谢！🙌
